integration/tests/nuxt/middleware.test.ts

```diff
@@ -0,0 +1,86 @@
+import { expect, test } from '@playwright/test';
+
+import type { Application } from '../../models/application';
+import { appConfigs } from '../../presets';
+import { createTestUtils } from '../../testUtils';
+
+test.describe('custom middleware @nuxt', () => {
```

Comment:
Let's add a package.json script for this then like:

https://github.com/clerk/javascript/blob/15ce5b110d754442ab92dd2454fbc973c4a66060/package.json#L45

Reply:
We have one but I'll move it next to vue script to make it easier to see 👍🏼 

https://github.com/clerk/javascript/blob/15ce5b110d754442ab92dd2454fbc973c4a66060/package.json#L39